scompose/project/project.py

```diff
@@ -431,7 +451,7 @@ def _create(
         # Generate ip addresses for each
         lookup = self.get_ip_lookup(names, bridge)
 
-        if self.sudo and not no_resolv:
+        if not no_resolv:
             # Generate shared hosts file
             hosts_file = self.create_hosts(lookup)
 
```

Comment:
The 'for loop' below prevents some containers from running. Any container that runs indefinitely will block execution of this loop and prevent all subsequent containers from running. Many web service containers operate this way.

I tried to pass command args like "indefinite.sh arg1 >& stdout.txt &" but I get errors related to invalid command args. Unless there's another existing method for backgrounding an indefinite execution script I'm not aware of, this loop ought to provide a new method to allow for this.